pkg/app/piped/platformprovider/lambda/client.go

```diff
@@ -27,9 +27,9 @@ import (
 	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
 	"github.com/aws/aws-sdk-go-v2/service/lambda"
 	"github.com/aws/aws-sdk-go-v2/service/lambda/types"
-	"go.uber.org/zap"
-
+	helper "github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
```

Comment:
Let's not use hashicorp retry but pipecd package 👀 
ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/backoff/backoff.go

Reply:
I use retry package to get lambda function state.